app/views/homepage/_more_on_govuk.html.erb

```diff
@@ -1,15 +1,17 @@
-<section class="homepage-more-on-govuk">
-  <%= render "govuk_publishing_components/components/heading", {
-    font_size: "m",
-    margin_bottom: 6,
-    text: t("homepage.index.more"),
-  } %>
-  <%= render "govuk_publishing_components/components/heading", {
-    text: t("homepage.index.most_active"),
-    margin_bottom: 4,
-    font_size: "s",
-    heading_level: 3,
-  } %>
+<section class="homepage-section homepage-more-on-govuk">
```

Comment:
Do we need that `homepage-more-on-govuk` class anymore?

Reply:
👍 yep don't need this anymore. Removed!